torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -13,7 +13,7 @@ namespace {
 
 // TODO This currently only handles tensors with requires_grad==False correctly.
 //      It should also handle autograd.
-Operator createOperatorFromC10(const c10::OperatorHandle& op) {
+Operator createOperatorFromC10_withTracingHandledHere(const c10::OperatorHandle& op) {
```

Comment:
It would be nice to have a brief comment about what the deal is with tracing handled/not handled here. Maybe just reuse text from lines 158/161/163